functions/public/styles/header.css

```diff
@@ -36,44 +19,6 @@ body{
     transition: all ease-in-out 400ms;
     margin: 0 auto;
 }
-a{
```

Comment:
All this deleted code has been transferred to the navigation file to be used for the navigation headers